packages/vouching/test/ZepToken.test.js

```diff
@@ -6,9 +7,13 @@ require('chai')
   .use(require('chai-bignumber')(BigNumber))
   .should();
 
-contract('ZepToken', ([ _, owner]) => {
-  beforeEach(async function () {
+contract('TPLToken', ([ _, owner, another, jurisdiction]) => {
+  const attributeID = 0;
+
+  beforeEach('deploy and initialize ZEP token', async function () {
     this.zepToken = await ZepToken.new();
```

Comment:
It's a good practice to avoid using the first address in the tests, as it is the default. Always be explicit about the `from` address.